packages/extensions/src/ext-meshopt-compression/meshopt-compression.ts

```diff
@@ -277,6 +278,14 @@ export class MeshoptCompression extends Extension {
 			// Example: https://skfb.ly/6qAD8
 			if (getTargetPath(accessor) === 'weights') continue;
 
+			// See: https://github.com/donmccurdy/glTF-Transform/issues/414
+			const isTexcoord = listSemantics(accessor, this.doc)
+				.some((semantic) => semantic.startsWith('TEXCOORD_'));
```

Comment:
If I omit this TEXCOORD_0 check and skip the unquantized NORMAL and TANGENT properties as well the output is fine... why do we need to skip more than TEXCOORD_*? And why are NORMAL and TANGENT not quantized?